cheroot/workers/dynpool.py

```diff
@@ -0,0 +1,277 @@
+# -*- coding: utf-8 -*-
+"""Dynamic pool.
+
+`dynpool <https://tabo.pe/projects/dynpool/>`_ is a Python
+library that handles the growing and shrinking of a pool
+of resources depending on usage patterns,
+written by `Gustavo Picón <https://tabo.pe/>`_ and
+licensed under the Apache License 2.0.
```

Comment:
I don't know if Apache 2.0 is compatible with BSD-3-Clause. You may need to acquire the permission for vendoring from @tabo.

Reply:
Removed dynpool vendoring.